src/epcpm/parameterstointerface.py

```diff
@@ -463,34 +499,34 @@ def gen(self):
             # TODO: CAMPid 9675436715674367943196954756419543975314
             getter_setter_list = [
                 "InterfaceItem",
-                var_or_func,
+                interface_type,
                 types[parameter.internal_type].name,
                 sunspec_type,
             ]
-
             sunspec_getter = "_".join(str(x) for x in getter_setter_list + ["getter"])
-            sunspec_setter = "_".join(str(x) for x in getter_setter_list + ["setter"])
+
+            if(interface_type != "constant"):
+                sunspec_setter = "_".join(str(x) for x in getter_setter_list + ["setter"])
+            #Constant is ReadOnly, doesnt need a setter
+            else:
```

Comment:
Since there are already other changes...  Is this a particular style preference vs. below?

```suggestion
            else:
                #Constant is ReadOnly, doesnt need a setter
```

Reply:
either works